--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -1,11 +1,5 @@
 # region Native

-[target.x86_64-unknown-linux-gnu]
-rustflags = ["-C", "link-arg=-fuse-ld=mold"]
-
-[target.aarch64-unknown-linux-gnu]
-rustflags = ["-C", "link-arg=-fuse-ld=mold"]
-
 [target.'cfg(all(windows, target_env = "msvc"))']
 rustflags = ["-C", "target-feature=+crt-static"]

@@ -40,39 +34,39 @@
 [target.mipsel-unknown-linux-musl]
 linker = "mipsel-unknown-linux-muslsf-gcc"
 rustflags = [
-    "-C",
-    "target-feature=+crt-static",
-    "-L",
-    "./musl_gcc/mipsel-unknown-linux-muslsf/mipsel-unknown-linux-muslsf/lib",
-    "-L",
-    "./musl_gcc/mipsel-unknown-linux-muslsf/mipsel-unknown-linux-muslsf/sysroot/usr/lib",
-    "-L",
-    "./musl_gcc/mipsel-unknown-linux-muslsf/lib/gcc/mipsel-unknown-linux-muslsf/15.1.0",
-    "-l",
-    "atomic",
-    "-l",
-    "ctz",
-    "-l",
-    "gcc",
+  "-C",
+  "target-feature=+crt-static",
+  "-L",
+  "./musl_gcc/mipsel-unknown-linux-muslsf/mipsel-unknown-linux-muslsf/lib",
+  "-L",
+  "./musl_gcc/mipsel-unknown-linux-muslsf/mipsel-unknown-linux-muslsf/sysroot/usr/lib",
+  "-L",
+  "./musl_gcc/mipsel-unknown-linux-muslsf/lib/gcc/mipsel-unknown-linux-muslsf/15.1.0",
+  "-l",
+  "atomic",
+  "-l",
+  "ctz",
+  "-l",
+  "gcc",
 ]

 [target.mips-unknown-linux-musl]
 linker = "mips-unknown-linux-muslsf-gcc"
 rustflags = [
-    "-C",
-    "target-feature=+crt-static",
-    "-L",
-    "./musl_gcc/mips-unknown-linux-muslsf/mips-unknown-linux-muslsf/lib",
-    "-L",
-    "./musl_gcc/mips-unknown-linux-muslsf/mips-unknown-linux-muslsf/sysroot/usr/lib",
-    "-L",
-    "./musl_gcc/mips-unknown-linux-muslsf/lib/gcc/mips-unknown-linux-muslsf/15.1.0",
-    "-l",
-    "atomic",
-    "-l",
-    "ctz",
-    "-l",
-    "gcc",
+  "-C",
+  "target-feature=+crt-static",
+  "-L",
+  "./musl_gcc/mips-unknown-linux-muslsf/mips-unknown-linux-muslsf/lib",
+  "-L",
+  "./musl_gcc/mips-unknown-linux-muslsf/mips-unknown-linux-muslsf/sysroot/usr/lib",
+  "-L",
+  "./musl_gcc/mips-unknown-linux-muslsf/lib/gcc/mips-unknown-linux-muslsf/15.1.0",
+  "-l",
+  "atomic",
+  "-l",
+  "ctz",
+  "-l",
+  "gcc",
 ]

 [target.aarch64-unknown-linux-ohos]
